can-queues/batch.js

```diff
@@ -0,0 +1,12 @@
+
+export default function transformer(file) {
+  let src = file.source;
```

Comment:
Where is this file coming from? Bad transforms.json generate?

Reply:
Might have made on accident. I'll remove.